scripts/scripts/testing/automations/incremental/nigeria.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from datetime import date
+
+
+def main():
+    data = pd.read_csv("automated_sheets/Nigeria.csv").sort_values(by="Date", ascending=False)
+
+    headers = {
+        "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"
+    }
+    source_url = "http://covid19.ncdc.gov.ng/"
+
+    req = requests.get(source_url, headers=headers)
+    soup = BeautifulSoup(req.content, "html.parser")
```

Comment:
You can try to use `cowidev` library here:

```python
soup = get_soup(source_url)
```

Note that for this to work you need to import `get_soup` function at the top of the script as:

```python
from cowidev.utils.web import get_soup
```